Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -21234,6 +21235,7 @@
 		0867D691FE84028FC02AAC07 /* WebCore */ = {
 			isa = PBXGroup;
 			children = (
+				33E03AFE2B59F557002992E3 /* EventNames.json */,
```

Comment:
This should be in `dom`, not the top-level `WebCore`.

Reply:
Hmm, I let Xcode do its thing... I'll try again!

Reply:
Fixed in 329f20a